### PR TITLE
feat(demo): demo add link to beta storybook

### PR DESCRIPTION
**Pull request by davidritter-dotcom**

### CODE REVIEW

**Review by gfellerph — CHANGES_REQUESTED** [review]
Added a little more rockets

File: packages/demo/src/app/home/home.component.html [hunk 1]
```diff
@@ -51,15 +51,32 @@ <h2 class="font-curve-large">Design System</h2>
...
           href="https://teams.microsoft.com/l/channel/19%3ae7fa68fb13eb40b4bf4604edea5f4b3e%40thread.tacv2/%25F0%259F%259A%2591%2520Support?groupId=123c7c9e-052a-40e6-98d3-6cc6d46bad0a&tenantId=3ae7c479-0cf1-47f4-8f84-929f364eff67"
           rel="noopener"
           target="_blank"
-        >Support chanel</a>
-        in Teams.</p>
+        >
+          Support chanel
+        </a>
+        in Teams.
+      </p>
```
[the remark this answers is not in the document]
Comment (davidritter-dotcom) [reply packages/demo/src/app/home/home.component.html:76]: I like the rocket, but is it supposed to go in the alert about Angular 16, shouldn't it be in the new alert about migreating towards storybook?

**Review by gfellerph — CHANGES_REQUESTED** [review]
1. 
image

The button is on the wrong alert.

2. There are 325 binary files from the angular compilation checked in (packages/components-angular/.angular/...) please untrack them https://stackoverflow.com/questions/18357511/git-remove-committed-file-after-push, please ask if you need help here.

**Review by gfellerph — CHANGES_REQUESTED** [review]
@davidritter-dotcom, please add a changeset for the demo "Added link to new documentation" or something along these lines.

**Review by gfellerph — APPROVED** [review]

**Pull request merged**